### PR TITLE
[new locale] ar-kw: Arabic (Kuwait) locale

### PR DESCRIPTION
**Pull request by nusretparlak**
Ar-Kw(Kuwait) Language Package Added .

### CODE REVIEW

File: locale/ar-kw.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        hh : '%d ساعات',
+        d : 'يوم',
+        dd : '%d أيام',
+        M : 'شهر',
+        MM : '%d أشهر',
+        y : 'سنة',
+        yy : '%d سنوات'
+    },
+    week : {
+        dow : 7, // Sunday is the first day of the week.
```
Comment (fbonzon) [inline locale/ar-kw.js]: `dow` should be from 0 to 6. Sunday is 0.

File: locale/ar-kw.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        doy : 12  // The week that contains Jan 1st is the first week of the year.
```
Comment (fbonzon) [inline locale/ar-kw.js]: The comment does not correspond to the actual value. Update the comment or delete it.

File: locale/ar-kw.js (hunk 1 above)
Comment (nusretparlak) [inline locale/ar-kw.js]: Day of week updated. Thanks for your suggest.

**Comment by ichernev** [comment]
You need `src/locale/ar-kw.js` and `src/test/locale/ar-kw.js`, other files are auto generated.

**Comment by nusretparlak** [comment]
I added locale file and test local file. thanks for your support.

**Comment by ichernev** [comment]
Merged in 63a0ad9a58b926b71923202248f81024f113f9b1

**Pull request closed without merging**